chapters/operatorsandexpressions.tex

```diff
@@ -739,6 +758,130 @@ \subsubsection{delay}\label{delay}
 \end{nonnormative}
 
 
+\subsubsection{spatialDistribution}\label{spatialdistribution}
+
+\begin{nonnormative}
+Many applications involve the modelling of variable-speed transport of properties.
+One option to model this infinite-dimensional system is to approximate it by an ODE, but this requires a large number of state variables and might introduce either numerical diffusion or numerical oscillations.
+Another option is to use a built-in operator that keeps track of the spatial distribution of $z(\xi, t)$, by suitable sampling, interpolation, and shifting of the stored distribution.
+In this case, the internal state of the operator is hidden from the ODE solver.
+\end{nonnormative}
+
+\lstinline!spatialDistribution! allows the infinite-dimensional problem below to be solved efficiently with good accuracy:
+\begin{align*}
+\frac{\partial z(\xi, t)}{\partial t} + v(t) \frac{\partial z(\xi, t)}{\partial \xi} &= 0\\
+z(0, t) &= \mathrm{in}_{0}(t) \quad \text{if $v(t) \geq 0$}\\
+z(1, t) &= \mathrm{in}_{1}(t) \quad \text{if $v(t) < 0$}
+\end{align*}
+where $z(\xi, t)$ is the transported quantity, $\xi$ is the normalized spatial coordinate ($0 \le \xi \le 1$), $t$ is the time, $v(t)$ is the normalized transport velocity and the boundary conditions are set at either $\xi = 0$ or $\xi = 1$, depending on the sign of the velocity.
+(When discontinuities are present, the differentiation operators above need to be understood in a distributive sense.)
+
+The calling syntax is:
+\begin{lstlisting}[language=modelica]
+(out0, out1) = spatialDistribution(in0, in1, x, positiveVelocity,
+                                   initialPoints = {0.0, 1.0},
+                                   initialValues = {0.0, 0.0});
+\end{lstlisting}
+where \lstinline!in0!, \lstinline!in1!, \lstinline!out0!, \lstinline!out1!, and \lstinline!x! are all subtypes of \lstinline!Real!, \lstinline!positiveVelocity! is a \lstinline!Boolean!, and \lstinline!initialPoints! and \lstinline!initialValues! are arrays of subtypes of \lstinline!Real!.
```

Comment:
I realize that I forgot to also lift the restriction to `Real`, so that we keep it as similar to `delay` as possible.  This was our intention, right?